### PR TITLE
AWS: bunch of improvements

### PR DESCRIPTION
**Pull request by xp-1000**
- use services metadata
- add org token
- update readme
- improve filtering behavior and doc for namespaces

### CODE REVIEW

File: cloud/aws/integrations-aws.tf [hunk 1]
```diff
@@ -1,12 +1,42 @@
-resource "signalfx_aws_external_integration" "sfx_integration_external" {
-  name = "AWSIntegration${var.suffix == "" ? "" : "-${title(var.suffix)}"}"
+locals {
```
Comment (Shr3ps) [inline cloud/aws/integrations-aws.tf:1]: not mandatory, but we like to extract locals constants to a locals.tf file

File: cloud/aws/integrations-aws.tf [hunk 2]
```diff
@@ -1,12 +1,42 @@
...
+  integration_name = "AWSIntegration${var.suffix == "" ? "" : "-${title(var.suffix)}"}"
+  open_aws_services = [
+    for service in data.signalfx_aws_services.aws_services.services.*.name:
```
Comment (Shr3ps) [inline cloud/aws/integrations-aws.tf:4]: .*. splat expression is deprecated in HCL2

**Review by Shr3ps — CHANGES_REQUESTED** [review]

**Comment by Shr3ps** [comment]
A changelog file with breaking changes warning would be nice

File: cloud/aws/integrations-aws.tf (hunk 2 above)
Comment (xp-1000) [reply cloud/aws/integrations-aws.tf]: done

File: cloud/aws/integrations-aws.tf (hunk 1 above)
Comment (xp-1000) [reply cloud/aws/integrations-aws.tf]: done

**Review by Shr3ps — APPROVED** [review]

**Pull request merged**